tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest-cov
 tox
+requests
```

Comment:
is this necessary? I can't find any uses of requests

Reply:
unfortunately yes... the fastapi / starlette testclient requires requests

Reply:
oh damn